src/scripts/scale.py

```diff
@@ -109,9 +109,13 @@ def method(base):
     assert f"unknown method: {ns.method}"
 
 
-# 0. check that the output doesn't exist
-assert not os.path.exists(ns.output_directory)
+# 0. check that the output doesn't exist if not performing in_place scaling
+assert os.path.exists(ns.output_directory) == ns.in_place
 store = zarr.DirectoryStore(ns.output_directory)
+if ns.in_place:
+    assert os.path.basename(os.path.normpath(ns.input_array)) in [
```

Comment:
Checks that input_array exists as an array of the output_directory group, if using in-place scaling.